python/packages/ai/teams/ai/prompts/prompt_manager.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, prompts_folder: str = "prompts") -> None:
     def add_function(
         self,
         name: str,
-        handler: Callable[[TurnContext, TurnState], Awaitable[str]],
+        handler: Callable[[TurnContext, TurnState], Awaitable[Any]],
```

Comment:
Is there any scenario that will return non-string value? The prompt template needs to replace corresponding variable with a string. There might be problem if developer does not return a string here?